driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/internal/BatchCursorFluxTest.java

```diff
@@ -282,4 +312,31 @@ private void assertCommandNames(final List<String> commandNames) {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * This method ensures that the server considers the specified {@code collection} existing for the purposes of, e.g.,
+     * {@link MongoCollection#drop()}, instead of replying with
+     * {@code "errmsg": "ns not found", "code": 26, "codeName": "NamespaceNotFound"}.
+     *
+     * @return {@code operationTime} starting at which the {@code collection} is guaranteed to exist.
+     */
+    private static BsonTimestamp ensureExists(final MongoClient client, final MongoCollection<Document> collection) {
+        BsonObjectId insertedId = Mono.from(collection.insertOne(Document.parse("{}")))
+                .map(InsertOneResult::getInsertedId)
+                .map(BsonValue::asObjectId)
```

Comment:
Observation: No need to cast to object id - an `_id` field can be of any BsonValue type.

Reply:
Thanks, removed the casting.